src/main/java/ro/msg/learning/shop/services/TestServiceImpl.java

```diff
@@ -0,0 +1,84 @@
+package ro.msg.learning.shop.services;
+
+import jakarta.transaction.Transactional;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.services.interfaces.TestService;
+import ro.msg.learning.shop.entitites.*;
+import ro.msg.learning.shop.repositories.*;
+import ro.msg.learning.shop.utils.TestingUtils;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class TestServiceImpl implements TestService {
```

Comment:
If it is a service only for testing purposes, please annotate it with the profile for test

Reply:
done.